multiaccounts/settings_notifications/database.go

```diff
@@ -0,0 +1,315 @@
+package notificationssettings
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type settingType int
+
+const (
+	stringType settingType = iota
+	integerType
+	boolType
+)
+
+// Columns' names
+type column string
+
+const (
+	columnTextValue          column = "text_value"
+	columnIntValue           column = "int_value"
+	columnBoolValue          column = "bool_value"
+	columnExMuteAllMessages  column = "ex_mute_all_messages"
+	columnExPersonalMentions column = "ex_personal_mentions"
+	columnExGlobalMentions   column = "ex_global_mentions"
+	columnExOtherMessages    column = "ex_other_messages"
+)
+
+// Static ids we use for notifications settings
+const (
+	keyAllowNotifications           = "AllowNotifications"
```

Comment:
@cammellos primary key is string cause for exemptions it will represent either community id, or 1:1 id, or group chat id and all of them are strings, leaded by that I created those constants which represent primary keys for the static notification settings (those known in advance, defined by design) and found it more readable using a string  for them than integer.

Also these constants starting with "key" are not part of enum, those are just constants.